src/game.h

```diff
@@ -462,8 +461,6 @@ class game
 
         item_location inv_map_splice( item_filter filter, const std::string &title, int radius = 0,
```

Comment:
`item_location_filter` should be the preferred variant although I'm not sure how much existing code would need to be updated to account for this?